src/twisted/python/test/test_url.py

```diff
@@ -5,12 +5,18 @@
 """
 Tests for L{twisted.python.url}.
 """
-
 from __future__ import unicode_literals
 
-from ..url import URL
+import warnings
+
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore", category=DeprecationWarning)
+    from ..url import URL
```

Comment:
The previous import was also relative.  This change's real intent is to suppress the `DeprecationWarning`.